Integration tests are failing on circle ci

After my latest commit https://github.com/mozilla/addons-frontend/commit/5a55cd35e7b9879fcb9d6095f628db69b623225e , integration tests are broken , 

```

=================================== FAILURES ===================================
__ test_firefox_footer_links[Resolution: 1080x1920-4-firefox/channel/desktop] __
[gw2] linux -- Python 3.6.9 /usr/bin/python3

base_url = 'http://olympia.test'
selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="77f5829d-def9-440f-b370-01d5898ce654")>
i = 4, links = 'firefox/channel/desktop'

    @pytest.mark.desktop_only
    @pytest.mark.parametrize(
        'i, links',
        # We enumerate the links in the "Browser" section of the footer.
        enumerate([
            'firefox/new',
            'firefox/mobile',
            'firefox/mobile',
            'firefox',
            'firefox/channel/desktop',
        ])
    )
    @pytest.mark.nondestructive
    def test_firefox_footer_links(base_url, selenium, i, links):
        page = Home(selenium, base_url).open()
>       page.footer.firefox_links[i].click()
E       IndexError: list index out of range

tests/frontend/ui/test_home.py:103: IndexError
------------------------------- pytest-selenium --------------------------------
Driver log: /tmp/pytest-of-seluser/pytest-1/popen-gw2/test_firefox_footer_links_Reso2/driver.log
URL: http://olympia.test/en-US/firefox/?locale=en-US
WARNING: Failed to gather log types: Message: HTTP method not allowed
______ test_firefox_footer_links[Resolution: 1080x1920-2-firefox/mobile] _______
[gw1] linux -- Python 3.6.9 /usr/bin/python3

base_url = 'http://olympia.test'
selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="b05e7ced-03b6-4bb9-95d3-e2fece4c6075")>
i = 2, links = 'firefox/mobile'

    @pytest.mark.desktop_only
    @pytest.mark.parametrize(
        'i, links',
        # We enumerate the links in the "Browser" section of the footer.
        enumerate([
            'firefox/new',
            'firefox/mobile',
            'firefox/mobile',
            'firefox',
            'firefox/channel/desktop',
        ])
    )
    @pytest.mark.nondestructive
    def test_firefox_footer_links(base_url, selenium, i, links):
        page = Home(selenium, base_url).open()
        page.footer.firefox_links[i].click()
>       assert links in selenium.current_url
E       assert 'firefox/mobile' in 'https://mixedreality.mozilla.org/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link'
E        +  where 'https://mixedreality.mozilla.org/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link' = <selenium.webdriver.firefox.webdriver.WebDriver (session="b05e7ced-03b6-4bb9-95d3-e2fece4c6075")>.current_url

tests/frontend/ui/test_home.py:104: AssertionError
------------------------------- pytest-selenium --------------------------------
Driver log: /tmp/pytest-of-seluser/pytest-1/popen-gw1/test_firefox_footer_links_Reso3/driver.log
URL: https://mixedreality.mozilla.org/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link
WARNING: Failed to gather log types: Message: HTTP method not allowed
```

--- a/tests/frontend/ui/pages/desktop/base.py
+++ b/tests/frontend/ui/pages/desktop/base.py
@@ -182,6 +182,6 @@ class Footer(Region):
         return header.find_elements(*self._footer_links)
 
     @property
-    def firefox_links(self):
+    def browsers_links(self):
         header = self.find_element(*self._footer_browsers_links)
         return header.find_elements(*self._footer_links)

--- a/tests/frontend/ui/test_home.py
+++ b/tests/frontend/ui/test_home.py
@@ -92,13 +92,13 @@ def test_add_ons_footer_links(base_url, selenium, i, links):
     enumerate([
         'firefox/new',
         'firefox/mobile',
-        'firefox/mobile',
+        'mixedreality.mozilla.org',
         'firefox',
         'firefox/channel/desktop',
     ])
 )
 @pytest.mark.nondestructive
-def test_firefox_footer_links(base_url, selenium, i, links):
+def test_browsers_footer_links(base_url, selenium, i, links):
     page = Home(selenium, base_url).open()
-    page.footer.firefox_links[i].click()
+    page.footer.browsers_links[i].click()
     assert links in selenium.current_url

